src/tasks/RhythmTask.py

```
@@ -0,0 +1,308 @@
+        endless = total == 0
+        count = 0
+
+        while endless or count < total:
+            count += 1
+            label = f"第 {count} 次" + ("" if endless else f"/{total}")
+
+            # 点击开始演奏
+            self.log_info(f"{label}：点击开始演奏")
+            self.operate_click(SONG_START_POS[0], SONG_START_POS[1])
+
+            # 等待离开选歌界面（最多15秒），确认进入音游后再开始检测
```

Ruff (0.15.12)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)